### ISSUE TITLE
Report Inactive Moderator Accounts

### ISSUE THREAD

**Issue by zmattingly** [jp00p/AGIMUS#418]
Per Gulliver Skidmore:

> I know Aggie can see when people are online. It may be worth automatically demoting someone who hasn't been seen at all (even lurking) for e.g. 6 months for security (abandoned accounts are more likely to get insecure as time goes on) and sending them a DM explaining why and saying let us know if you are back and we'll re-promote you

**Comment by rahmcoff** [comment]
Probably the easiest way to do this is to add a `last_modified` to the `user` table.  Since it would update on any XP, and mouthing you do adds XP, we could use that instead of tracking logins.  This would miss lurkers, but do we want to keep lurkers at high rank?

**Comment by bhammerslag** [comment]
No, lurker accounts should not have elevated privileges for the reasons given in the suggestion. If someone is a Commander or whatever and loses it for inactivity, they can be reinstated by request or whatever. 

Don't see any harm in letting people keep Ensign though

**Comment by magikid** [comment]
> Since it would update on any XP

Isn’t it possible to opt out of the XP system though?

**Comment by rahmcoff** [comment]
I had thought that it was impossible to opt out of the XP, just that it stop level ups and badge trading.  My memory was that there were users that had opted out, and when they opted in again, they got several levels in a row.  But looking at the actual thing in `xp.py`, it does require it be enabled to earn any XP.
